apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -0,0 +1,216 @@
+/**
+ * Generated by `createschema meter.MeterReading 'number:Integer; date:DateTimeUtc; account?:Relationship:BillingAccount:SET_NULL; billingAccountMeter?:Relationship:BillingAccountMeter:SET_NULL; meter:Relationship:Meter:CASCADE; property:Relationship:Property:CASCADE; organization:Relationship:Organization:CASCADE; value:Integer; sectionName?:Text; floorName?:Text; unitName?:Text; client?:Relationship:User:SET_NULL; clientName?:Text; clientEmail?:Text; clientPhone?:Text; contact?:Relationship:Contact:SET_NULL; source:Relationship:MeterSource:PROTECT'`
+ */
+
+const { Text, Relationship, Integer, DateTimeUtc } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/meter/access/MeterReading')
+const { getLastBillingAccountMeterReading } = require('../utils/serverSchema')
+const { VALUE_LESS_THAN_PREVIOUS_ERROR } = require('@condo/domains/meter/constants/errors')
+const { ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
+const { AutoIncrementInteger } = require('@core/keystone/fields')
+const { PHONE_WRONG_FORMAT_ERROR } = require('@condo/domains/common/constants/errors')
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+
+
+const MeterReading = new GQLListSchema('MeterReading', {
+    schemaDoc: 'Meter reading taken from a client or billing',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        number: {
```

Comment:
? is it really needed

Reply:
No, this is the old logic for combining readings. Now it is not needed, removed